projects/js-packages/publicize-components/src/hooks/use-media-restrictions/constants.ts

```diff
@@ -0,0 +1,33 @@
+import { __ } from '@wordpress/i18n';
+
+export const NO_MEDIA_ERROR = 'NO_MEDIA_ERROR';
+export const FILE_TYPE_ERROR = 'FILE_TYPE_ERROR';
```

Comment:
Just double-checking that we consciously skipped this from the error messages below to favor the generic `Invalid media`

Reply:
Yes, I think that message is appropriate for it.